src/schema/Transfer/dataloaders/Events.ts

```diff
@@ -1,78 +0,0 @@
-/**************************************************************************
- *  (C) Copyright Mojaloop Foundation 2020                                *
- *                                                                        *
- *  This file is made available under the terms of the license agreement  *
- *  specified in the corresponding source code repository.                *
- *                                                                        *
- *  ORIGINAL AUTHOR:                                                      *
- *       Yevhen Kyriukha <yevhen.kyriukha@modusbox.com>                   *
- **************************************************************************/
-
```

Comment:
Why this file is removed?

Reply:
This file used the eventStoreMongo client for event lookups, which we are no longer using. 
I have created a new dataloaders file for lookups using the Prisma client.